.github/CONTRIBUTING.md

```diff
@@ -1,388 +1,98 @@
 # Contributing
 
-## Requirements
+## Prerequisites
 
-Set up your SSH Key GitHub account and install node.js 4 or higher.
+To contribute to Carbon, you'll need to install:
 
-- [Generating SSH Keys - GitHub](https://help.github.com/articles/generating-ssh-keys/)
-- [`nvm` (Node Version Manager)](https://github.com/creationix/nvm) to use the
-  `Node 6`.
+- [node(v10 or higher)](https://nodejs.org/en/download/)(you can download it
+  here or follow their installation through a package manager
+  [here](https://nodejs.org/en/download/package-manager/))
+  - We recommend using
+    [nvm](https://github.com/nvm-sh/nvm/blob/master/README.md) as your version
+    manager for Node.
+- [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
```

Comment:
Should we share a link for `git`? This pathway is a hard one to predict given how many people use machines that have it pre-loaded 😬

Reply:
I agree. Plus, should be `Git`.

Reply:
Would you recommend just putting `git` or suggesting they check to see if they have it installed, telling them how, and if not supplying this link?

Reply:
Changed it to just Git with no link, should be good now 👍